File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/mediaplayer/PlaylistController.java
package com.kendoui.spring.controllers.mediaplayer;

import java.util.ArrayList;
import java.util.Locale;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import com.kendoui.spring.models.Video;

@Controller("mediaplayer-playlist-controller")
@RequestMapping(value = "/mediaplayer/")
public class PlaylistController {

    @RequestMapping(value = "playlist", method = RequestMethod.GET)
    public String index(Locale locale, Model model) {

        ArrayList<Video> videos = new ArrayList<Video>();   
        videos.add(new Video("Digital Transformation: A New Way of Thinking","https://www.youtube.com/watch?v=gNlya720gbk", "http://img.youtube.com/vi/gNlya720gbk/1.jpg"));
        videos.add(new Video("Learn How York Solved Its Database Problem","https://www.youtube.com/watch?v=_S63eCewxRg", "http://img.youtube.com/vi/_S63eCewxRg/1.jpg"));
        videos.add(new Video("Responsive Website Delivers for Reeves Import Motorcars","https://www.youtube.com/watch?v=DYsiJRmIQZw", "http://img.youtube.com/vi/DYsiJRmIQZw/1.jpg"));
        videos.add(new Video("Telerik Platform - Enterprise Mobility. Unshackled.","https://www.youtube.com/watch?v=N3P6MyvL-t4", "http://img.youtube.com/vi/N3P6MyvL-t4/1.jpg"));
        videos.add(new Video("Take a Tour of the Telerik Platform","https://www.youtube.com/watch?v=rLtTuFbuf1c", "http://img.youtube.com/vi/rLtTuFbuf1c/1.jpg"));
        videos.add(new Video("Why Telerik Analytics - Key Benefits for your applications","https://www.youtube.com/watch?v=cIOHxCcsvXA", "http://img.youtube.com/vi/cIOHxCcsvXA/1.jpg"));
        
        model.addAttribute("videos", videos);

        return "mediaplayer/playlist";
    }
}

File: wrappers/mvc-6/demos/Kendo.Mvc.Examples/Controllers/MediaPlayer/PlaylistController.cs
using Kendo.Mvc.Examples.Models;
using Kendo.Mvc.Extensions;
using Kendo.Mvc.UI;
using Microsoft.AspNetCore.Mvc;
using System.Collections.Generic;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class MediaPlayerController : Controller
    {
        public ActionResult Playlist()
        {
            return View(GetVideos());
        }

        public ActionResult Videos_Read([DataSourceRequest] DataSourceRequest request)
        {
            return Json(GetVideos().ToDataSourceResult(request));
        }

        private static IEnumerable<Video> GetVideos()
        {
            List<Video> videos = new List<Video>();

            videos.Add(new Video() { title = "Digital Transformation: A New Way of Thinking", poster = "http://img.youtube.com/vi/gNlya720gbk/1.jpg", source = "https://www.youtube.com/watch?v=gNlya720gbk" });
            videos.Add(new Video() { title = "Learn How York Solved Its Database Problem", poster = "http://img.youtube.com/vi/_S63eCewxRg/1.jpg", source = "https://www.youtube.com/watch?v=_S63eCewxRg" });
            videos.Add(new Video() { title = "Responsive Website Delivers for Reeves Import Motorcars", poster = "http://img.youtube.com/vi/DYsiJRmIQZw/1.jpg", source = "https://www.youtube.com/watch?v=DYsiJRmIQZw" });
            videos.Add(new Video() { title = "Telerik Platform - Enterprise Mobility. Unshackled.", poster = "http://img.youtube.com/vi/N3P6MyvL-t4/1.jpg", source = "https://www.youtube.com/watch?v=N3P6MyvL-t4" });
            videos.Add(new Video() { title = "Take a Tour of the Telerik Platform", poster = "http://img.youtube.com/vi/rLtTuFbuf1c/1.jpg", source = "https://www.youtube.com/watch?v=rLtTuFbuf1c" });
            videos.Add(new Video() { title = "Why Telerik Analytics - Key Benefits for your applications", poster = "http://img.youtube.com/vi/cIOHxCcsvXA/1.jpg", source = "https://www.youtube.com/watch?v=cIOHxCcsvXA" });

            return videos;
        }
    }
}
